MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,428 @@
+        this.dataForm.DefaultLayoutSettings.LabelWidth = new DataFormItemLength(400, DataFormItemLengthUnitType.Absolute);
+
+{% endhighlight %}
+{% highlight C# %}
+
+        this.dataForm.DefaultLayoutSettings.LabelWidth = new DataFormItemLength(0.3, DataFormItemLengthUnitType.Star);
+        this.dataForm.DefaultLayoutSettings.LabelWidth = new DataFormItemLength(0.7, DataFormItemLengthUnitType.Star);
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Label width customization is applicable only when `LabelPosition` is Left.
+
+By default, the available width is divided into 40% for label and 60% for editor.
+
+## Spanning editors rows and columns
+
+Row height and column width can be increased by defining the `DataFormDisplayOptions` attribute or by handling `GenerateDataFormItem` event.
+
+### Row span
+
+#### Using attributes
+
+Row height can be increased by using the [RowSpan] property in the`DataFormDisplayOptions` attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataFormDisplayOptions(RowSpan = 2)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+The row height of each `DataFormItem` can also be increased by using the [RowSpan] property and it will be handled in the `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.RowSpan = 2;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, `Name` field’s row height is increased.
+
+### Column span
+
+#### Using attributes
+
+When the grid layout is used, the column width can be increased by using the [ColumnSpan] property in the `DataFormDisplayOptions` attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.ColumnCount = 2;
+
+{% endhighlight %}
+{% highlight C# %}
+
+    [DataFormDisplayOptions(ColumnSpan = 2)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+When the grid layout is used, you can increase the column width of each `DataFormItem` using the [ColumnSpan] property and it will be handled in the `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.ColumnSpan = 2;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Change the editor visibility
+
+The data form editor visibility can be changed at run time by using the [IsVisible] of `DataFormItem`.
```

Comment:
The data form editor visibility can be changed  by using the [IsVisible] of `DataFormItem`.

Reply:
Content changed